docs/registries.md

```diff
@@ -16,16 +16,22 @@ To support private GitHub, simply authenticate with your private GitHub account:
 ```
 
 ```
-Would you like to set up your GitHub credentials? [yes]: 
+Would you like to set up your GitHub credentials? [yes]:
      If using two-factor authentication or to avoid using your password you can generate an access token at https://github.com/settings/applications.
 
 Enter your GitHub username: username
-Enter your GitHub password or access token: 
-Would you like to test these credentials? [yes]: 
+Enter your GitHub password or access token:
+Would you like to test these credentials? [yes]:
 ```
 
 This will enable private repo installs.
 
+#### Github Rate Limiting
```

Comment:
Right, I just don't want people to confuse this for normal authentication which should be communicated to users first via `jspm registry config github`. That's also why this should be lower down with a clear title that shows it is specific to automated workflows.